javascript/react/basics-ii/conditional-rendering-part-1.md

```diff
@@ -34,7 +34,7 @@ aspects:
 
 **Conditional rendering** is the long-standing technique of `render`ing different `element`s for different scenarios.
 
-Exploiting the power of pure `JS`, **React** allows conditional rendering with simple `if`s or *ternary operators*[1].
+Exploiting the power of pure JS, **React** allows conditional rendering with simple `if`s or *ternary operators*[1].
```

Comment:
```suggestion
Exploiting the power of pure JS, **React** allows conditional rendering with `if`s or *ternary operators*[1].
```

Reply:
The revision q bellow should be more explicit that conditional rendering technique we speak about here are not created by React and just common within it.

Something like:

> Which JS capabilities are commonly used in **React** for **conditional rendering**?